### ISSUE TITLE
Melee

### ISSUE THREAD

**Issue by Drew12473** [zbot473/SurvivHacks#90]
Sometimes when you play, you can't equip the melee weapon but sometimes you can.

**Comment by YESS808** [comment]
Drew I'm guessing that has to do with AutoSwitch, try disabling that feature and see if you can use melee, I disabled autoswitch and autoheal because they keep stopping me from doing what I want to do lol. (Middle of close quarters battle... reload... *Using Pills* "NO! <X> <R>eload.." ... *Using Pills* You are dead.) lol.

**Comment by alexvornoffice** [comment]
Autoswitch keeps enabling every-time even if I disable it, strange.

**Comment by zbot473** [comment]
hmm

**Comment by zbot473** [comment]
@YESS808 @Kalaborative Reinstall the extension completely.

**Comment by YESS808** [comment]
Yeah I tried that several times, even manually deleted C:\Users\USERNAME\AppData\Local\Google\Chrome\User Data\Default\Local Extension Settings\ifncjjnkgiciepckagaefbcfejbikpjo where the extension was storing the config, it erased my other custom settings but AutoSwitch still turns itself back on every time the extension loads.

**Comment by zbot473** [comment]
It is because the script detects dead ppl
Also reinstalling should fix the toggle bug
